1.3/spec.md

```diff
@@ -0,0 +1,222 @@
+[version 3.0.0](http://sqlite.org/formatchng.html) or higher.
+Only core SQLite features are permitted; tilesets **cannot require extensions**.
+
+MBTiles databases can optionally use [the officially assigned magic number](http://www.sqlite.org/src/artifact?ci=trunk&filename=magic.txt)
+to be easily identified as MBTiles.
+
+## Database
+
+Note: the schemas outlined are meant to be followed as interfaces.
+SQLite views that produce compatible results are equally valid.
+For convenience, this specification refers to tables and virtual
+tables (views) as tables.
+
+### Metadata
+
+#### Schema
+
+The database is required to contain a table or view named `metadata`.
+
+This table must yield exactly two columns named `name` and
+`value`. A typical create statement for the `metadata` table:
+
+    CREATE TABLE metadata (name text, value text);
+
+It is common to create an index for this table:
+
+    CREATE UNIQUE INDEX name on metadata (name);
+
+#### Content
+
+The metadata table is used as a key/value store for settings. Two keys are **required**:
+
+* `name`: The plain-English name of the tileset.
+* `format`: The file format of the tile data.
+
+For new tilesets, the `format` should be a MIME type: for example,
+`image/png` or `image/jpeg` for PNG or JPEG image data, or
+`application/vnd.mapbox-vector-tile` for
+[Mapbox Vector Tile](https://github.com/mapbox/vector-tile-spec/) data.
+
+Previous versions of the MBTiles spec called for PNG and JPEG data
+to be tagged with a `format` of `png` or `jpg`, so these may be found
+in existing tilesets and should be treated as synonyms for
+`image/png` and `image/jpeg`. Many existing Mapbox Vector Tile tilesets
+are tagged with a `format` of `pbf`, which should be treated as
+a synonym for `application/vnd.mapbox-vector-tile`.
+
+Four rows in `metadata` are **suggested** and, if provided, may enhance performance:
+
+* `bounds`: The maximum extent of the rendered map area. Bounds must define an
+  area covered by all zoom levels. The bounds are represented in `WGS:84` -
+  latitude and longitude values, in the OpenLayers Bounds format -
+  **left, bottom, right, top**. Example of the full earth: `-180.0,-85,180,85`.
+* `center`: The longitude, latitude, and zoom level of the default view of the map.
+  Example: `-122.1906,37.7599,11`
+* `minzoom`: The lowest zoom level for which the tileset provides data
+* `maxzoom`: The highest zoom level for which the tileset provides data
+
+Other optional rows encode more information about the tileset:
+
+* `attribution`: An attribution string, which explains in English (and HTML) the sources of
+  data and/or style for the map.
+* `description`: A description of the tileset's content as plain text.
+* `type`: `overlay` or `baselayer`
+* `version`: The version of the tileset, as a plain number.
+  This refers to a revision of the tileset itself, not of the MBTiles specification.
+* `scheme`: `tms`, to make the tiling scheme explicit
```

Comment:
Good point. I mentioned this only because someone [filed a ticket asking for it](https://github.com/mapbox/tippecanoe/issues/237), but I should not be inventing fields when my only goal is to document what existing implementations do.